sdk/core/abort-controller/package.json

```diff
@@ -101,7 +101,7 @@
     "nyc": "^14.0.0",
     "prettier": "^1.16.4",
     "rimraf": "^2.6.2",
-    "rollup": "~1.13.1",
+    "rollup": "^1.0.0",
```

Comment:
Should the accepted version start at `1.16.3` since that's the minimum (recent) version of rollup that won't break `@azure/event-hubs`?

So something like `^1.16.3` should allow any version that is >= 1.16.3 and less than 2.x

Reply:
I agree, ^1.16.3 seems reasonable to me. Since this is a dev dep too, don't need to be super expansive. Requiring latest (or close to it) seems reasonable when we're fixing bugs left and right :)

Reply:
Good point.  I will update the PR but not run tests again, since the effective version is the same (`1.16.6`).